docs/intro.rst

```diff
@@ -271,7 +271,7 @@ running:
 
 If you installed Debian or RPM packages, you can simply run ``kafka-rest-start``
```

Comment:
unrelated to this change: we should probably mention the systemd init scripts here.

Reply:
@edenhill do you think this needs mentioning in each individual component where this is available, e.g. Kafka, Schema-Registry, C3, and Kafka Connect? Related, see https://github.com/confluentinc/docs/pull/764/files.